config/augmentation/basic_augmentation_segmentation.yaml

```diff
@@ -11,7 +11,7 @@ augmentation:
 
   # Augmentations done immediately before conversion to torch tensor
   normalization: # Normalization: parameters for finetuning. See examples below:
-    mean:  # -> mean: [0.485, 0.456, 0.406]
-    std:  # -> std: std: [0.229, 0.224, 0.225])
+    mean: [1e-8, 1e-8, 1e-8] # -> mean: [0.485, 0.456, 0.406]
+    std: [1e-8, 1e-8, 1e-8] # -> std: std: [0.229, 0.224, 0.225])
```

Comment:
These are just default values with no meaning; it is there because some tests rely on the default values. You're right, do not use for actual experiments

Reply:
I believe, [0.485, 0.456, 0.406] is a set of mean values for the ImageNet dataset

Reply:
@valhassan I understand it is needed for testing, but aren't the transformation loaded by default in our training script? Maybe we could set stdevs values to 1 to be sure they have no impact if a user don't define them. For the mean I guess it would have almost no impact as it is.

@ms888ekb I think you are right, I found that information too.

Reply:
most of pretrained model available from torch use these values as default normalization (mean and std)
it's ok to change them as long as we don't rely on pretrained torch models

Reply:
Understood.

The new placeholder values will be 

mean: [0.0, 0.0, 0.0]
std: [1, 1, 1]

This should have no effect on the inputs